tests/test_records/test_validator.py

```diff
@@ -0,0 +1,131 @@
+from harvester.records.validators import ValidateGeoshapeWKT
+
+
+def test_validator_envelope_returns_original_value_success(
+    caplog,
+):
+    caplog.set_level("DEBUG")
+
+    class TestValidatedXMLSourceRecord:
```

Comment:
Personal preference but I find defining classes in tests a bit cluttered and this:
```
    class TestValidatedXMLSourceRecord:
        @ValidateGeoshapeWKT
```

 is very repetitive, can this be a fixture in `conftest.py` and then modify the value for each test?